open_humans/management/commands/expunge_project_data_files.py

```diff
@@ -0,0 +1,28 @@
+import arrow
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from private_sharing.models import ProjectDataFile
+
+
+class Command(BaseCommand):
+    """
+    A management command for expunging incomplete project data files.
```

Comment:
At the risk of creating redundant and/or stale docstrings, I'd like to see a bit more context provided. Like what makes a data file incomplete? When are things expunged?

In this case, maybe something like this?

ProjectDataFile objects which are not complete (complete=False) represent incomplete upload attempts. (See ViewReference1 and ViewReference2.) This command deletes objects that have been incomplete for longer than the period specified by settings.INCOMPLETE_DATA_FILE_EXPIRATION.